Makefile

```diff
@@ -7,7 +7,7 @@ START=goFile
 TEST_DIR=codegen/tests
 
 define run-sysl
-sysl codegen --dep-path github.com/anz-bank/sysl-go-comms/$(TEST_DIR)/  --root . --root-transform . --transform $< --grammar $(GRAMMAR) --start $(START) --outdir $(OUT) --app-name $(APP) $(MODEL)
+sysl codegen --dep-path github.com/anz-bank/sysl-go/$(TEST_DIR)/  --root . --root-transform . --transform $< --grammar $(GRAMMAR) --start $(START) --outdir $(OUT) --app-name $(APP) $(MODEL)
```

Comment:
Make is broken: `malformed module path "github.com/anz-bank/sysl-go-comms/codegen/tests//callback"`, not sure how it happened but it looks like `$(EXT_LIB_DIR)` fell off the end: `--dep-path github.com/anz-bank/sysl-go/$(TEST_DIR)/$(EXT_LIB_DIR)`.

Reply:
@gkanz Yeah struggling to fix the make and test issues, I think that fixed a whole bunch of errors, thanks!

Reply:
Fixed in https://github.com/anz-bank/sysl-go/pull/14/commits/a531f931ff08bc4ea2227e0f2e9907bb444abc48